metadata_tools/index_support.py

```
@@ -524,20 +539,83 @@ def process_index(template_name, glob=None):
 
-                unused = index.unused if not unused else unused & index.unused
+                    index.create(labels_only=labels_only, pattern=pattern)
+                    unused = index.unused if not unused else unused & index.unused
+
+        # Write the task file
+        if task_list_only:
+            com.write_task_file(task_file)
 
         # Log a warning for any columns that never had non-null values
         if unused:
             logger.warn('Unused columns: %s', unused)
```

Ruff (0.14.10)

Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)